backport-Guide.md

```diff
@@ -0,0 +1,84 @@
+# Kata Containers stable backport workflow
+
+* [Backport Workflow](#backport-workflow)
+
+This document provides a short guide on how to perform a stable backport of a
+PR or patchset in the Kata Containers repositories.
+
+It does not cover all eventualities that may be encountered whilst performing
+a backport, and only documents one potential git based workflow, but should
+serve as a short introduction and reminder of the basic process.
+
+## Backport Workflow
+
+The basic workflow involves creating a new local branch from the stable tree you
+are targetting, then cherry picking the commits from your master branch PR
+onto that branch. You then submit your branch to github as a PR against the
+stable branch (and not the `master` branch).
+
+The following example shows a pseudo-backport of a PR from the `master` branch
+into the `stable-1.2` branch of the `runtime` repo.
+
+- Ensure your local repo is upto date:
+
+```bash
+$ cd ${GOPATH}/src/github.com/kata-containers/runtime
+$ git fetch upstream
+```
+
+- Create your branch to work on
+```bash
+$ git checkout -b my_1.2_pr_backport origin/stable-1.2
+```
+
+- Locate the commits you wish to pull in
+
+Here we are looking in the `master` branch, presuming the PR has already been merged,
+but if you have the PR in a local branch you can substitute `master` for the name
+of that branch here.
+
+```bash
+$ git log --oneline master
+# And search for the SHA of the commits you wish to backport.
+```
+
+- Pull in your commits
+
+If you are pulling them in from the `master` branch, then you can add the `-x`
+argument to `git cherry-pick` to automatically add a reference in the
+commit to the original commits. This is strongly recommended to aid traceability.
+If you are picking the commits from your local branch tnen do *not* use `-x`, as this
+will potentially add references to SHAs that only exist in your user branches, and thus
+will not be useful for future tracability.
+
+It is also required you add the `-s` signoff to the commits, if the original
+commits did not originate from yourself.
+
+```bash
+$ git cherry-pick -s -x <commit>...
+```
+
+You can cherry pick ranges of commits etc. Please see the `git-cherry-pick(1)`
+man page for more information.
+
+- Resolve any conflicts etc.
+
+You may hit conflicts during your cherry pick, which will need fixing before
+you can continue. Follow standard practice for git conflict resolution, and see
+the guidance printed by `git cherry-pick` on how to process and apply those fixes.
+
+If you hit a conflict, any effects of `-x` to `cherry-pick` may not be
+applied. In this case, consider hand-adding the `master` SHA references and a note
+that you have had to resolve a conflict to the commit message.
+
+- Push your branch to your github repo
```

Comment:
Before this, please add a step to ask the user to test their branch since even if there is no git conflict, there is no guarantee the code will even build ;)